Misc/reputations.json

```diff
@@ -44,9 +45,10 @@
       "regex": "\\b(?:(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\[\\.\\]){3}(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\b",
       "reputationCommand": "ip",
       "details": "IP",
+      "contextValue": "Address",
+      "contextPath": "IP(val.Address && val.Address === obj.Address)",
       "enhancementScriptNames": [
         "IPReputation",
-        "SplunkSearch",
         "SplunkPySearch"
```

Comment:
this script is deprecated, should be removed from here

Reply:
Leaving for backward compatibility